MAUI/Calendar/getting-started.md

```diff
@@ -7,23 +7,45 @@ control: SfCalendar
 documentation: ug
 ---
 
-# Getting started with .NET MAUI Calendar (SfCalendar)
+# Getting Started with the .NET MAUI Calendar
+
 This section explains how to add the [.NET MAUI Calendar](https://www.syncfusion.com/maui-controls/maui-calendar) control. This section covers only the basic features needed to get started with Syncfusion Calendar.
 
```

Comment:
Follow the steps below to add a basic scheduler view to your project.

- Add this line

Reply:
Added